easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2025.04-GCCcore-14.2.0.eb

```diff
@@ -0,0 +1,533 @@
+easyblock = 'CargoPythonBundle'
+
+name = 'Python-bundle-PyPI'
+version = '2025.04'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages from PyPI"
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('UnZip', '6.0'),
+    ('pkgconf', '2.3.0'),
+    ('git', '2.49.0'),  # required for pbr
+    ('hatchling', '1.27.0'),
+    ('poetry', '2.1.2'),
+    ('scikit-build-core', '0.11.1'),  # required for rapidfuzz
+    ('flit', '3.10.1'),
+    ('Rust', '1.85.1'),
+    ('setuptools-rust', '1.11.0'),  # required for dulwich
+    ('maturin', '1.8.3'),  # required for rpds-py
+]
+
+dependencies = [
+    ('Python', '3.13.1'),
+    ('cryptography', '44.0.2'),
+    ('virtualenv', '20.29.2'),
+]
+
+crates = [
```

Comment:
@Micket For what are these required?

I'm hoping we can avoid requiring Rust for `Python-bundle-PyPI` (although I'm not sure how long we can keep that up).

Reply:
Any idea how painful it would be to pull those out (and includes them as dependencies in this bundle)?

Reply:
Well, probably not to bad for these two, just a question of how many of the extensions gets pulled out i suppose.

But do you want us to go back and rip these out of Python-bundle-PyPI-2024.06-GCCcore-13.3.0.eb as well?
Just changing it to a CargoPythonBundle would be slightly less intrusive for those who already built it.

Reply:
I think we concluded in the zoom meeting that we'll just go with this as is rather than trying to break things out, since we depend (indirectly) on Rust anyway via cryptography.

Reply:
Indeed